README.md

```diff
@@ -1,146 +1,272 @@
+**Fondant is the platform where we meet to build and share data preparation workflows.**
 
-```
-pip install git+https://github.com/ml6team/fondant.git
-```
+Fondant offers a framework to build **composable data preparation pipelines, with reusable 
+components, optimized to handle massive datasets**. Stop building from scratch, and start 
+leveraging reusable components instead to:
+- Extend your data with public datasets
+- Generate new modalities using captioning, segmentation, translation, image generation, ...
+- Distill knowledge from existing foundation models
+- Filter out low quality data
+- Deduplicate data
 
-## Introduction
+And create high quality datasets to fine-tune your own foundation models.
 
-Fondant is built upon [KubeFlow pipelines](https://www.kubeflow.org/docs/components/pipelines/), a cloud-agnostic and open-source framework built by Google to run machine learning workflows on Kubernetes.
+<p align="right">(<a href="#chocolate_bar-fondant">back to top</a>)</p>
 
-Fondant consists of a pipeline that runs a sequence of components (steps) one after the other. Each step implements one logical piece of a data processing pipeline, like image filtering, image embedding, text deduplication, etc.
+## :magic_wand: Example pipelines
 
-A Fondant pipeline always includes a loading component as first component, followed by one or more transform components. The loading component loads some initial seed data, with the transform components making transformations on them (like enrichment, filtering, transformation).
+Curious to see what Fondant can do? Have a look at our example pipelines:
 
-## Fondant Components
+### Fine-tuning ControlNet
+
+Our 
+[example pipeline to generate data for ControlNet fine-tuning](examples/pipelines/controlnet-interior-design) 
+allows you to create models that you can control using inpainting, segmentation, and 
+regeneration. All you need to get started is a set of prompts describing the type of images to 
+generate.
+
+Using our ControlNet model fine-tuned on interior design images for instance allows you to generate 
+the room of your dreams:
+
+| Input image                                                    | Output image                                                     |
+|----------------------------------------------------------------|------------------------------------------------------------------|
+| ![input image](docs/art/interior_design_controlnet_input1.png) | ![output image](docs/art/interior_design_controlnet_output1.jpg) |
+
+Want to try out the resulting model yourself, head over to our 
+[Hugging Face space](https://huggingface.co/spaces/ml6team/controlnet-interior-design)!
+
+### Fine-tuning Stable Diffusion
+
+Using our 
+[example pipeline to fine-tune Stable Diffusion](examples/pipelines/finetune_stable_diffusion) 
+allows you create models that generate better images within a specific domain. All you need to 
```

Comment:
typo, missing "to": allows you "to" create models that generate better images within a specific domain.